common/internal_model/src/main/scala/uk/ac/wellcome/models/work/internal/DeletedReasons.scala

```diff
@@ -0,0 +1,8 @@
+package uk.ac.wellcome.models.work.internal
+
+sealed trait DeletedReason
+
+object DeletedReason {
+  case class DeletedFromSource(info: String) extends DeletedReason
+  case class SuppressedFromSource(info: String) extends DeletedReason
+}
```

Comment:
Really nice to have these in the index.

Do we need to do something to ensure they're not searchable, invisible fields?

Reply:
they are invisible cause the works that carry them are not visible. We want them to be searchable cause they are searched by people in #collections-info

Reply:
Yeah, I’ve realised what I asked doesn't make sense. 😅